### ISSUE TITLE
Would you consider open-sourcing your work?

### ISSUE THREAD

**Issue by gcampax** [Kitt-AI/snowboy#651]
Very sad to hear that you're going away.

It makes sense that technology developed 4 years ago is no longer state of the art, in this rapidly changing field. Yet, snowboy works well from an engineering perspective: it's simple to install and to use, and uses relative small amounts of training data.
Given that, would you consider open-sourcing the library (including the blob) and backend?
That would be enormously helpful for the community, which currently has no high-quality open-source wake-word implementation.

**Comment by Naveen3Singh** [comment]
Or at least the dataset used for the previous models?  Especially 'Jarvis'

**Comment by Alagunto** [comment]
Agreed, open-sourcing the backend after the site's shutdown would be a huge joy for small indie developers. Hope my +1 commend would drag some attention to this thread :)

**Comment by Sammysedo** [comment]
Open-sourcing the backend would be huge news for small developers as it would significantly lengthen the support for high-quality, offline speech recognition and wake word training after the platform shuts down. Please reply to this @chenguoguo
